sdk/metricsadvisor/perf-tests/ai-metrics-advisor/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
You should probably align with the storage tests.  @HarshaNalluru would know best.

https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/perf-tests/storage-blob/tsconfig.json

Reply:
Storage required more entries like "dom" in the lib and probably target ES5(aligned with what it had at its package level).

The current config in this PR aligns with the rest of the packages.. like TA, form-recognizer, etc.
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/perf-tests/text-analytics/tsconfig.json
Looking good.

Reply:
Yeah actually he suggested that storage tests had to use special settings for tsconfig, and that I should use text analytics as reference - https://github.com/Azure/azure-sdk-for-js/pull/14288#discussion_r594731433